traitsui/examples/demo/Advanced/Popup_Dialog_demo.py

```diff
@@ -2,6 +2,8 @@
 #  License: BSD Style.
 
 """
+Note: This demo only works on the wx backend.
```

Comment:
There is a `Notes` section at the bottom. Should this go there or do you think this is important enough and therefore needs to go at the top?

Reply:
I think its fine in the bottom section, I will move it.